src/Claimer.sol

```diff
@@ -51,35 +68,81 @@ contract Claimer is Multicall {
   }
 
   /// @notice Allows the call to claim prizes on behalf of others.
-  /// @param vault The vault to claim from
-  /// @param tier The tier to claim for
-  /// @param winners The array of winners to claim for
-  /// @param prizeIndices The array of prize indices to claim for each winner (length should match winners)
+  /// @param _vault The vault to claim from
+  /// @param _tier The tier to claim for
+  /// @param _winners The array of winners to claim for
+  /// @param _prizeIndices The array of prize indices to claim for each winner (length should match winners)
   /// @param _feeRecipient The address to receive the claim fees
+  /// @param _minVrgdaFeePerClaim The minimum fee for each claim
   /// @return totalFees The total fees collected across all successful claims
   function claimPrizes(
-    Vault vault,
-    uint8 tier,
-    address[] calldata winners,
-    uint32[][] calldata prizeIndices,
-    address _feeRecipient
+    Vault _vault,
+    uint8 _tier,
+    address[] calldata _winners,
+    uint32[][] calldata _prizeIndices,
+    address _feeRecipient,
+    uint256 _minVrgdaFeePerClaim
```

Comment:
I don't understand this parameter.
If I provide `_minVrgdaFeePerClaim` when calling `claimPrizes`, can't I just pass `0` and the condition `(feePerClaim < _minVrgdaFeePerClaim)` will always be false?

Reply:
Yes that's right; if the minimum is zero then it'll always work just fine.

Does that make sense?